frontend/src/features/Vessel/components/VesselList/DownloadVesselListModal.tsx

```diff
@@ -85,28 +85,26 @@ export function DownloadVesselListModal({ filteredVessels, isOpen, setIsOpen })
       .filter(vessel => vessel.checked)
       .map(vessel => {
         // eslint-disable-next-line no-param-reassign
-        vessel.vesselProperties.latitude = getCoordinates(
-          vessel.coordinates,
-          OPENLAYERS_PROJECTION,
-          coordinatesFormat
-        )[0]?.replace(/ /g, '')
+        vessel.latitude = getCoordinates(vessel.coordinates, OPENLAYERS_PROJECTION, coordinatesFormat)[0]?.replace(
+          / /g,
+          ''
+        )
         // eslint-disable-next-line no-param-reassign
-        vessel.vesselProperties.longitude = getCoordinates(
-          vessel.coordinates,
-          OPENLAYERS_PROJECTION,
-          coordinatesFormat
-        )[1]?.replace(/ /g, '')
+        vessel.longitude = getCoordinates(vessel.coordinates, OPENLAYERS_PROJECTION, coordinatesFormat)[1]?.replace(
+          / /g,
+          ''
+        )
 
         const filteredVesselObject: Record<string, string> = {}
         checkboxState.valuesChecked.forEach(valueChecked => {
           switch (valueChecked) {
             case CSVOptions.flagState.code:
-              filteredVesselObject[CSVOptions.flagState.code] = vessel?.vesselProperties?.flagState
-                ? (countries.getName(vessel?.vesselProperties?.flagState, 'fr') ?? '')
+              filteredVesselObject[CSVOptions.flagState.code] = vessel?.flagState
+                ? (countries.getName(vessel?.flagState, 'fr') ?? '')
                 : ''
               break
             default: {
-              const value = vessel[valueChecked] || vessel?.vesselProperties[valueChecked] || ''
+              const value = vessel[valueChecked] || ''
```

Comment:
```suggestion
              const value = vessel[valueChecked] ?? ''
```

Je ne suis pas sûr de comprendre le `value.toString()` en-dessous du coup.

Reply:
Le type de `vessel[valueChecked]` peut ne pas être une string